{
  "comments": [
    {
      "key": {
        "uuid": "7d00a622_2744b2e2",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 5206
      },
      "writtenOn": "2020-09-21T18:59:31Z",
      "side": 1,
      "message": "\u003e 2 of 22 SlowBots failed:\n\u003e Failed on solaris-amd64-oraclerel: https://storage.googleapis.com/go-build-log/d04a6feb/solaris-amd64-oraclerel_6c8bbad7.log\n\u003e Failed on illumos-amd64: https://storage.googleapis.com/go-build-log/d04a6feb/illumos-amd64_49a4da93.log\n\u003e \n\u003e Consult https://build.golang.org/ to see whether they are new failures. Keep in mind that TryBots currently test *exactly* your git commit, without rebasing. If your commit\u0027s git parent is old, the failure might\u0027ve already been fixed.SlowBot builds that ran:\n\u003e * illumos-amd64\n\u003e * solaris-amd64-oraclerel\n\n\nI don\u0027t see why the test is failing on illumos.",
      "revId": "d04a6feb65751b642e1356c0718bd7c7b4743b4a",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "28fcb7b4_32a60678",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 19560
      },
      "writtenOn": "2020-09-22T07:20:33Z",
      "side": 1,
      "message": "I don\u0027t understand the failure either. On my local illumos VM this works fine. Maybe something specific with the builder?",
      "parentUuid": "7d00a622_2744b2e2",
      "revId": "d04a6feb65751b642e1356c0718bd7c7b4743b4a",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "d423d521_2b337947",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 5206
      },
      "writtenOn": "2020-09-22T21:39:47Z",
      "side": 1,
      "message": "I can fix the problem if I force the misc/cgo/errors ptrtest test source to import \"net\", which it doesn\u0027t otherwise do.  Importing \"net\" brings in net/cgo_solaris.go, which adds -lsocket -lnsl -lsendfile to the external link line.  So the problem is that the syscall package refers to accept4, but doesn\u0027t do anything to link against -lsocket.",
      "parentUuid": "28fcb7b4_32a60678",
      "revId": "d04a6feb65751b642e1356c0718bd7c7b4743b4a",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "52b94aca_2c162870",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 19560
      },
      "writtenOn": "2020-09-22T22:25:22Z",
      "side": 1,
      "message": "Thanks. I\u0027ve fixed this by adding\n\n  //go:cgo_import_dynamic _ _ \"libsocket.so\"\n\nto syscall/syscall_illumos.go. Not sure if this is the correct solution, but testing this on the illumos builder using gomote seems to pass the misc/cgo/errors tests now.",
      "parentUuid": "d423d521_2b337947",
      "revId": "d04a6feb65751b642e1356c0718bd7c7b4743b4a",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705",
      "unresolved": true
    }
  ]
}